src/sentry/static/sentry/app/views/organizationDashboard/exploreWidget.jsx

```diff
@@ -81,42 +75,96 @@ class ExploreWidget extends React.Component {
     const discoverQueries = widget.queries.discover;
 
     return (
-      <DropdownLink
-        anchorRight={true}
-        title={t('Explore')}
-        topLevelClasses={exploreMenuCss}
-      >
-        {discoverQueries.map(query => (
-          <ExploreRow key={query.name}>
-            <QueryName>{query.name}</QueryName>
-
-            <ExploreActions>
-              <Button borderless size="zero" to={this.getExportToDiscover(query)}>
-                <InlineSvg src="icon-discover" />
-              </Button>
-              <Button borderless size="zero" to={this.getExportToEvents(query)}>
-                <InlineSvg src="icon-stack" />
-              </Button>
-            </ExploreActions>
-          </ExploreRow>
-        ))}
-      </DropdownLink>
+      <DropdownMenu>
+        {({isOpen, getRootProps, getActorProps, getMenuProps}) => {
+          return (
+            <ExploreRoot {...getRootProps()}>
+              <div {...getActorProps()}>
```

Comment:
This `div` is needed unfortunately to get DropdownMenu working well (otherwise it won't toggle to close). `emotion` makes this complicated because of `ref`s, I believe upgrading to emotion@10 may solve this.